### PR TITLE
Object conversion and imports

### PR DESCRIPTION
**Pull request by rngomez303**
Imports were cleaned up to fit correct formatting. Functions for converting objects to dictionary form for json output were added to objects needed for Gsyslyzer output.

### CODE REVIEW

File: src/constants.py [hunk 1]
```diff
@@ -5,8 +5,7 @@
 class VerbosityLevel(enum.IntEnum):
     """ Define levels of output verbosity"""
     LOW = 0
-    MEDIUM = 1
-    HIGH = 2
+    HIGH = 1
```
Comment (jadmanski) [inline src/constants.py:8]: No MEDIUM anymore? I guess if it's not used.

File: src/group_parser.py [hunk 2]
```diff
@@ -121,11 +121,21 @@ def detect_bursts(self, detected_signals, confirmed_signals):
...
-            burst = SymptomBurst(burst_list)
+            burst = symptom_burst.SymptomBurst(burst_list)
             self.bursts.append(burst)
 
             if prev_symptom_tag not in self.burst_dict:
                 self.burst_dict[prev_symptom_tag] = [burst]
             else:
                 self.burst_dict[prev_symptom_tag].append(burst)
 
+    def convert_self_to_dict(self):
```
Comment (jadmanski) [inline src/group_parser.py]: The "self" in the name is probably not that helpful; a convert_to_dict method with no parameters would generally be implied to be operating on self.

**Review by jadmanski — COMMENTED** [review]
Should this include changes to actually use the new conversion functions?

File: src/constants.py (hunk 1 above)
Comment (rngomez303) [reply src/constants.py:8]: I didn't have a use for it yet, so I figured it should be omitted until there is a proper use.

File: src/group_parser.py (hunk 2 above)
Comment (rngomez303) [reply src/group_parser.py]: Ok - I can change the function name.

**Comment by rngomez303** [comment]
The file that has the code that uses these functions is part of the other pending pull request, but that one has a commit on it with the changes to use the new functions.

**Review by jadmanski — APPROVED** [review]

**Review by dragisa-google — APPROVED** [review]

**Pull request merged**